packages/mdl-tooltip/README.md

```diff
@@ -0,0 +1,39 @@
+# MDL Tooltip
+
+The MDL Tooltip component is a spec-aligned tooltip component adhering to the
+[Material Design tooltip requirements](https://material.google.com/components/tooltips.html).
+It works without JavaScript.
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+## Usage
+
+To use the tooltip component add the class `mdl-tooltip` to any element.
+Then provide that element an `aria-label` attribute to define what the contents of the tooltip should be.
+
+```html
+<!-- Using the material icons font for a favorite icon. -->
+<i class="material-icons mdl-tooltip" aria-label="Favorite">favorite</i>
+```
+
+## Developer Notes
+
+The tooltip takes advantage of the `::after` pseudo element.
+Tooltips can only be made with elements that do not use this themselves.
+To make your components work with tooltips take advantage of only `::before` within your component code.
```

Comment:
Alternatively, they could use a sentinel element and a combo of a unique id + `aria-labelledby` + `aria-label` on the tooltip sentinel.

```html
<style>
  .tooltip-wrapper {
    position: relative;
  }

  .tooltip-wrapper > .mdl-tooltip {
    position: absolute;
    width: 100%;
    height: 100%;
    z-index: -1; /* maybe? */
  }
</style>
<div class="tooltip-wrapper">
  <i class="material-icons" aria-labelledby="icon-tooltip">favorite</i>
  <div class="mdl-tooltip" aria-label="Save to Favorites"></div>
</div>
```

_or_, we could refactor the mdl-tooltip base styles into either a base css class, or a sass modifier, so that they can be applied to other elements.